Spec/RestClient.swift

```diff
@@ -1173,12 +1173,15 @@ class RestClient: QuickSpec {
                             testHTTPExecutor.http = MockHTTP(network: caseTest, logger: options.logHandler)
                             let channel = client.channels.get("test")
 
+                            let lock = NSLock()
                             testHTTPExecutor.afterRequest = { _, callback in
+                                lock.lock()
```

Comment:
I would prefer to see this acquisition constrained, so it would perhaps be healthier to consider a sensible timeout and fail the test if that's exceeded via [`lock(before:)`](https://developer.apple.com/documentation/foundation/nslock/1411133-lock).

Reply:
Done f6196d7. Thanks